tests/ui/pages/modals/add-tag-modal.ts

```diff
@@ -0,0 +1,52 @@
+import InputField from "../../elements/input-field";
+import Button from "../../elements/button";
+import { Pages } from "../index";
+import BaseModal from "./base-modal";
+
+const SELECTORS = {
+    this_dialog: 'div[role="dialog"]',
+    tag_name: '[placeholder="Tag Name"]',
+    add_tag: '[type="submit"]',
+    important_tag: '[type="checkbox"]',
+    add_one_more_tag: 'div[role="dialog"] button:has-text("Create tag")',
+    tag_line_root: 'div[class*="MuiGrid-root MuiGrid-container css-1d3bbye"]',
+
+}
+
+export default class AddTagModal extends BaseModal {
+    constructor(pages: Pages) {
+        super(pages, SELECTORS.this_dialog);
+    }
+
+    get tag_name_field() {
+        return new InputField(this.page, SELECTORS.tag_name);
+    }
+
+    get add_new_tag() {
+        return new Button(this.page, SELECTORS.add_tag);
+    }
+
+    /**
+     *
+     * @param i - index of checkbox
+     */
+    async check_important(i: number) {
```

Comment:
use human-readable naming

Reply:
Done

Reply:
can we rename argument (`i`) to smth more meaningful?

For instance, `indexOfCheckbox`.